File: Mobile/media_tracker_test/lib/models/tmdb/tmdb_tv_show.dart
class TMDBTvShow {
  final int id;
  final String name;
  final String overview;

  TMDBTvShow({required this.id, required this.name, required this.overview});

  factory TMDBTvShow.fromJson(Map<String, dynamic> json) {
    return TMDBTvShow(
      id: json['id'],
      name: json['name'],
      overview: json['overview'],
    );
  }
}

File: Desktop/media-tracker-desktop/media-tracker-desktop/Models/TMDB/TMDB_TV_Show.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace media_tracker_desktop.Models.TMDB
{
    // This model represents tv show object.
    public class TMDB_TV_Show
    {
        [JsonProperty("id")]
        public int ID { get; }

        [JsonProperty("name")]
        public string Name { get; } = string.Empty;

        [JsonProperty("overview")]
        public string Overview { get; } = string.Empty;

        [JsonConstructor]
        public TMDB_TV_Show(int id, string name, string overview)
        {
            this.ID = id;
            this.Name = name;
            this.Overview = overview;
        }
    }
}
